csrankings-4.csv

```diff
@@ -2134,6 +2134,7 @@ Krothapalli Sreenivasa Rao,IIT Kharagpur,https://cse.iitkgp.ac.in/~ksrao,TATEL2E
 Krung Sinapiromsaran,Chulalongkorn University,http://pioneer.netserv.chula.ac.th/~skrung,Pqic3LgAAAAJ
 Krys J. Kochut,University of Georgia,http://cobweb.cs.uga.edu/~kochut,NOSCHOLARPAGE
 Krys Kochut,University of Georgia,http://cobweb.cs.uga.edu/~kochut,NOSCHOLARPAGE
+Krzakala Florent,EPFL,https://florentkrzakala.com/

```

Comment:
Adding the Google Scholar entry as well (https://scholar.google.com/citations?user=3jDeUlMAAAAJ)
```suggestion
Krzakala Florent,EPFL,https://florentkrzakala.com/,3jDeUlMAAAAJ
```

Reply:
This faculty member is in Physics, not CS.

Reply:
I am terribly sorry, but I respectfully disagree: you cannot be more wrong.

In EPFL, I am in 50% Engineering & 50% in Physics. Here is the source:
- https://actu.epfl.ch/news/two-new-professor-nominations-at-sb/

- I am also affiliated with the CS department for the doctoral program: Here is the source https://www.epfl.ch/education/phd/edic-computer-and-communication-sciences/edic-computer-and-communication-sciences/edic-affiliated-labs-research-expertise/
- https://people.epfl.ch/florent.krzakala

Finally, I do believe that after publishing in Neurips (where i serve currently as an Area chair and was the top European contributor last year), COLT, ICML, SODA, ITW, IEEE, STOC, etc...  it feels weird to be rejected as a "physicist" !